languages/wikipron/src/whitelist.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+import sys
+import re
+
+
+def whitelist_reader(path):
+    """Reads in whitelist file"""
+    with open(path, "r") as source:
+        for line in source:
+            yield line.rstrip().split()[0]
```

Comment:
Please type this function. It should be:

   `def whitelist_reader(path: str) -> Iterator[str]:`

Is this splitting to separate off the comment character? Feels a bit non-robust. I would instead suggest using a regular expression substitution (`re.sub`) to remove the comments.